src/util/uint.rs

```diff
@@ -91,6 +91,27 @@ macro_rules! construct_uint {
             }
         }
 
+        impl PartialOrd for $name {
```

Comment:
Can we use `::std::cmp` for consistency here and in the next impl?

Reply:
We could, but it's not an exported macro, so it's not very crucial.

Reply:
Well if you didn't notice that's a copy-paste impl from https://github.com/rust-bitcoin/rust-bitcoin/pull/356/files#diff-ca925ea633ee82c8b9dfd44534146107L114
anyhow as Steven said these are internal macros so we(aka Andrew ;) ) never cared much about explicit here because they're just use in this crate, and all these traits are included the in the prelude (https://doc.rust-lang.org/std/prelude/index.html)

Reply:
I have to export these macros in my develop branch of the rust-bitcoin (b/c I use them in libs depending on rust-bitcoin), and we may decide to make them exportable in the future.

Reply:
I really don't think we should export these. It is not the job of rust-bitcoin to be a general-purpose bignum (or whatever) library.

Reply:
Sure, I'm not asking for that. I just do export in my own fork of rust-bitcoin :) But anyway it would be good to stick to the same code style within the macro itself, even if it is internal